source/titback/backend.cpp

```diff
@@ -3,25 +3,67 @@
  * See /LICENSE.md for license information. SPDX-License-Identifier: MIT
 \* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
 
+#include <cstring>
 #include <filesystem>
 #include <string>
+#include <utility>
 
 #include <crow/app.h>
 #include <crow/http_request.h>
 #include <crow/http_response.h>
+#include <cxxopts.hpp>
 
+#include "tit/core/io.hpp"
 #include "tit/core/main_func.hpp"
 #include "tit/core/sys/utils.hpp"
 
+#include "titback/interpreter.hpp"
+
 namespace tit::back {
 namespace {
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-auto run_backend(CmdArgs /*args*/) -> int {
+auto run_backend(CmdArgs args) -> int {
+  // Parse the command line arguments.
+  cxxopts::Options options("titback", "Backend for the Tit Solver.");
```

Comment:
`cxxopts` does not seem like the best choice so far. We should either design our own implementation for the command line arguments parsing, or make some sort of a wrapper.

Reply:
Let's right now parse the arguments manually. We'll create a proper parser later.